ports/geckolib/gecko_bindings/structs_debug.rs

```diff
@@ -2792,7 +2753,7 @@ impl ::std::clone::Clone for nsIRequest {
 #[repr(C)]
 #[derive(Debug, Copy)]
 pub struct EventStates {
-    pub mStates: ::std::os::raw::c_ulong,
+    pub mStates: ::std::os::raw::c_ulonglong,
```

Comment:
Unfortunately not right now, bindgen resolves typedefs inside classes to avoid other (arguably worst) problems. We could stop if we find some known type, I'll fill an issue about that.